Goal: Recommend using adduser and deluser.

Fixes: #406046

Status wrt upstream: Debian specific patch.

--- a/man/useradd.8.xml
+++ b/man/useradd.8.xml
@@ -84,6 +84,12 @@
   <refsect1 id='description'>
     <title>DESCRIPTION</title>
       <para>
+	<command>useradd</command> is a low level utility for adding
+	users.  On Debian, administrators should usually use
+	<citerefentry><refentrytitle>adduser</refentrytitle>
+	<manvolnum>8</manvolnum></citerefentry> instead.
+      </para>
+      <para>
 	When invoked without the <option>-D</option> option, the
 	<command>useradd</command> command creates a new user account using
 	the values specified on the command line plus the default values from
--- a/man/userdel.8.xml
+++ b/man/userdel.8.xml
@@ -64,6 +64,12 @@
   <refsect1 id='description'>
     <title>DESCRIPTION</title>
     <para>
+      <command>userdel</command> is a low level utility for removing
+      users.  On Debian, administrators should usually use
+      <citerefentry><refentrytitle>deluser</refentrytitle>
+      <manvolnum>8</manvolnum></citerefentry> instead.
+    </para>
+    <para>
       The <command>userdel</command> command modifies the system account
       files, deleting all entries that refer to the user name <emphasis
       remap='I'>LOGIN</emphasis>. The named user must exist.
